gpu4pyscf/dft/rks.py

```diff
@@ -217,6 +217,12 @@ def energy_elec(ks, dm=None, h1e=None, vhf=None):
     e1 = cupy.einsum('ij,ji->', h1e, dm).real
     ecoul = vhf.ecoul.real
     exc = vhf.exc.real
+    if isinstance(ecoul, cupy.ndarray):
```

Comment:
`.get()` method returns a 0-dimensional array. It's better to change these values to scalar variables, using `float(ecoul)` or `ecoul.get()[()]`